src/discord/discord.service.ts

```diff
@@ -80,7 +104,9 @@ export class DiscordService implements OnModuleInit {
   }
 
   async getChannels() {
-    const guild = await this.client.guilds.fetch('990712194842902629');
+    const guild = await this.client.guilds.fetch(
+      this.configService.getOrThrow('DISCORD_SERVER_ID'),
```

Comment:
I like the pattern you have with doing this config fetching in a property instead of mixed in with the logic. Want to do the same here?

Reply:
Done!